test/test_jit_fuser.py

```diff
@@ -816,6 +826,9 @@ def fn_test_erf(x):
 
     @unittest.skipIf(not RUN_CUDA, "fuser requires CUDA")
     def test_rand_broadcast_cuda(self):
+        if graph_executor_mode() != ProfilingMode.PROFILING:
+            self.skipTest("Passes only with profiling executor")
```

Comment:
why did it stop passing with the legacy executor?

Reply:
I have no idea, but we never actually tested it for the reasons I mentioned previously.